# Request from Client (sensor_data_node) 
bool request
---
# Response from Server (teensy node)
uint64 bat
uint64 inf_left
uint64 inf_middle
uint64 inf_right
